documentation/smg/04_building_with_cmake.md

```diff
@@ -1,38 +1,151 @@
 # Building with CMake
 
+## Build dependencies
+
+A developer building Herbert/Horace will need to install:
+
+- [Matlab](https://www.mathworks.com/products/matlab.html) >= 2018b
+  - on Linux the desired Matlab version must be on the path when running the
+  script
+  - on Windows, by default, the latest version is found via the registry
+- [CMake](https://cmake.org/download/) >= 3.7
+  - must be on the path for both Windows and Linux
+- System compiler:
+  - Linux: [GCC](https://gcc.gnu.org/) >= 6.3.0
+  - Windows:
+  [Visual Studio build tools](https://visualstudio.microsoft.com/downloads/#build-tools-for-visual-studio-2019)
+  \>= 2017
+
+## Using build scripts
+
+Horace and Herbert contain PowerShell and Bash scripts named `build.[ps1|sh]`.
+These scripts provide an easy way to build Herbert/Horace on Windows or Linux.
+Theses same scripts are used to build Horace and Herbert on the CI servers,
+so they are tested regularly.
+
```

Comment:
Again, checking the likely audience. Guessing CI is not meaningful to external users or even non-developers.